library/azure_rm_webapp_slot.py

```diff
@@ -0,0 +1,993 @@
+        default: started
+
+    state:
+      description:
+        - Assert the state of the Web App deployment slot.
+        - Use 'present' to create or update a deployment slot and 'absent' to delete it.
+      default: present
+      choices:
+        - absent
+        - present
+
+extends_documentation_fragment:
+    - azure
+    - azure_tags
+
+author:
+    - "Yunge Zhu(@yungezz)"
+
+'''
+
+EXAMPLES = '''
+    - name: Create a windows web app with non-exist app service plan
+      azure_rm_webapp:
+        resource_group: myresourcegroup
+        name: mywinwebapp
+        plan:
+          resource_group: myappserviceplan_rg
+          name: myappserviceplan
+          is_linux: false
+          sku: S1
+
+    - name: Create a docker web app with some app settings, with docker image
+      azure_rm_webapp:
+        resource_group: myresourcegroup
+        name: mydockerwebapp
+        plan:
+          resource_group: appserviceplan_test
+          name: myappplan
+          is_linux: true
+          sku: S1
+          number_of_workers: 2
+        app_settings:
+          testkey: testvalue
+          testkey2: testvalue2
+        container_settings:
+          name: ansible/ansible:ubuntu1404
+
+    - name: Create a docker web app with private acr registry
+      azure_rm_webapp:
+        resource_group: myresourcegroup
+        name: mydockerwebapp
+        plan: myappplan
+        app_settings:
+          testkey: testvalue
+        container_settings:
+          name: ansible/ubuntu1404
+          registry_server_url: myregistry.io
+          registry_server_user: user
+          registry_server_password: pass
+
+    - name: Create a linux web app with Node 6.6 framework
+      azure_rm_webapp:
+        resource_group: myresourcegroup
+        name: mylinuxwebapp
+        plan:
+          resource_group: appserviceplan_test
+          name: myappplan
+        app_settings:
+          testkey: testvalue
+        frameworks:
+          - name: "node"
+            version: "6.6"
+
+    - name: Create a windows web app with node, php
+      azure_rm_webapp:
+        resource_group: myresourcegroup
+        name: mywinwebapp
+        plan:
+          resource_group: appserviceplan_test
+          name: myappplan
+        app_settings:
+          testkey: testvalue
+        frameworks:
+          - name: "node"
+            version: 6.6
+          - name: "php"
+            version: "7.0"
+
+    - name: Create a linux web app with java framework
+      azure_rm_webapp:
+        resource_group: myresourcegroup
+        name: mylinuxwebapp
+        plan:
+          resource_group: appserviceplan_test
+          name: myappplan
+        app_settings:
+          testkey: testvalue
+        frameworks:
+          - name: "java"
+            version: "8"
+            settings:
+              java_container: "Tomcat"
+              java_container_version: "8.5"
+'''
+
+RETURN = '''
+azure_webapp:
+    description: Id of current web app.
```

Comment:
shouldn't "id" be just on the top?

i mean "azure_webapp" is a dict, and description is "Id of current web app."

Reply:
self.results contains "id", so there's no "azure_webapp" dict

Reply:
fixed